bedrock/firefox/templates/firefox/welcome/page11.html

```diff
@@ -38,7 +38,7 @@ <h1 class="c-main-title">{{ ftl('welcome-page11-main-heading') }}</h1>
 
       <div class="mzp-c-picto">
         <div class="mzp-c-picto-image">
-          {% include 'firefox/welcome/includes/page11/globe.svg' %}
+          <img src="{{ static('img/firefox/welcome/page11/globe.svg') }}" alt="">
```

Comment:
**issue:** It looks like this page supports both light and dark theme, so when viewed in dark mode it's hard to see the individual image lines. I'd suggest changing these to use `<picture>`, which can then serve light / dark versions of each image when needed using `(prefers-color-scheme: light/dark)`. You can see the page in prod for reference: https://www.mozilla.org/de/firefox/welcome/11/

Reply:
Holy *! I've checked it out and thought it was as designed, but now see the `invert-fill` class for it from the document level. I'll need to think a bit about this, as I usually just "hue-rotate" the hell out of it via CSS but this is def more involved. Very smart execution though.

Will explore using `fill="currentColor"` for the blacks and setting the dark mode appearance via color on the DOM parent, will see how that goes… Thanks for catching this!

Reply:
Any alternative solutions are welcome here fwiw. I believe this page has probably seen _most_ of its traffic, so it's only worth making the effort to migrate the images as they were if it's not too much effort.

Reply:
This one was fun;) Of course that `currentColor` also has the usual document boundary limit so it won't inherit the colour from a surrounding DOM it's not a part of…:/

But luckily, the classes and media prefers were already there, just needed moving inside the SVGs 😇

Reply:
FYI this has a caveat with some Safari versions not applying the correct color-scheme to child documents like here (i.e. the actual SVG opened just itself as a top document follows light/dark; but loaded inside a page that prefers light/dark won't switch the atmedia rule:/…)

But given this is an in-product page, I assumed Gecko compatibility is sufficient.

Reply:
Yeah i think we're OK here seeing as this is a Firefox only page. Typically we'd use a `<picture>` element here instead, but what you have works for this use case. Thanks!

Reply:
Never realised there would be so much quirks even today. _Learning something new every day along the way… ;)_

Reply:
This gotcha is a new one for me too fwiw, TIL!